cmd/openstack-populator/openstack-populator.go

```diff
@@ -232,3 +184,138 @@ func writeData(reader io.ReadCloser, file *os.File, imageID string, progress *pr
 
 	return nil
 }
+
+func getAuthType() (authType clientconfig.AuthType, err error) {
+	if configuredAuthType := getStringFromSecret(AuthType); configuredAuthType == "" {
+		authType = clientconfig.AuthPassword
+	} else if supportedAuthType, found := supportedAuthTypes[configuredAuthType]; found {
+		authType = supportedAuthType
+	} else {
+		err = errors.New(UnsupportedAuthTypeErrStr)
+		klog.Fatal(err.Error(), "authType", configuredAuthType)
+	}
+	return
+}
+
+func getStringFromSecret(key string) string {
+	value, err := os.ReadFile(fmt.Sprintf("/etc/secret-volume/%s", key))
+	if err != nil {
+		klog.Fatal(err.Error())
```

Comment:
```
2023-06-30T08:43:34.089974467Z stderr F F0630 08:43:34.088028       1 openstack-populator.go:203] open /etc/secret-volume/availability: no such file or directory

```
I suppose this should klog.Error or klog.Info

Reply:
`klog.Info` I would say. I will change it.